package.json

```diff
@@ -10,6 +10,7 @@
     "detoxTestIOS": "detox build -c ios.sim.debug && detox test -c ios.sim.debug",
     "reduxDebugger": "remotedev-debugger --hostname localhost --port 5678 --injectserver",
     "lint": "node_modules/eslint_d/bin/eslint.js src __tests__ --cache",
+    "lintJson": "node_modules/.bin/jsonlint -q locales/en.json",
```

Comment:
Should this check only `locales/en.json` or also all the other files that will eventually be in that folder such as `fr.josn`...?

Other than this I agree that it is a good idea to add test in travis.

Reply:
I only check `en.json` because it's the only one that should be manually modified by devs. The other languages are sent over by transifex, so I expect them to be valid json. That said, it could be worth checking them as well, just in case someone adds a manual fix. I'll open a separate PR.